store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -63,6 +63,14 @@ private void instantiatePolicy() {
      * @throws UnsupportedEncodingException
      */
     public String sanitize() throws UnsupportedEncodingException {
+        return processSanitization(true).toString();
+    }
+
+    public String sanitize(boolean cleanData) throws UnsupportedEncodingException {
```

Comment:
Sanitizing is cleaning up data, can we rename **cleanData** to something a little more verbose, like **cleanMalformedHtml**?